taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    country = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=50, unique=True)
+    password = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.username
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=50)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name='cars'
+    )
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='cars')
+
+    def __str__(self):
+        return f"CAR:{self.model}, manufacturer:{self.manufacturer}, drivers:{self.drivers}"
```

Comment:
Code style issue: You should define the `Meta` class for the `Car` model. This class is used to provide metadata for the model. It should contain `verbose_name` and `verbose_name_plural` attributes in lowercase. For example:

class Meta:
    verbose_name = "car"
    verbose_name_plural = "cars"